e0works/MyDiary-front
# #161938368 users should be able to modify a diary entry

#### What does this PR do?
- implement update diary entry

#### Description of Task to be completed?
- write test
- add update entry action and reducer
- add updateEntry method on Dashboard component


#### How should this be manually tested?
- start the app
- click on the edit button on any entry
- modify the entry and click update
- you should see an updated version of the diary entry

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#161938368](https://www.pivotaltracker.com/story/show/161938368, "view single entry")

#### Screenshots (if appropriate)
NA
#### Questions:
